app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/TransactionDetailsBottomSheetPresenter.kt

```diff
@@ -1,104 +0,0 @@
-            is TransactionDetailsLaunchState.Id -> handleId(state)
-        }
-    }
-
-    private fun handleId(state: TransactionDetailsLaunchState.Id) {
-        launch {
-            try {
-                view?.showLoading(true)
-
-                val details = historyInteractor.getHistoryTransaction(state.tokenPublicKey)
-
-                handleHistory(details)
-            } catch (e: Throwable) {
-                Timber.e(e, "Error loading transaction details")
-                view?.showError(R.string.details_transaction_not_found)
-            } finally {
-                view?.showLoading(false)
-            }
-        }
-    }
-
-    private fun handleHistory(transaction: HistoryTransaction) {
-        when (transaction) {
-            is HistoryTransaction.Swap -> parseSwap(transaction)
-            is HistoryTransaction.Transfer -> parseTransfer(transaction)
-            is HistoryTransaction.BurnOrMint -> parseBurnOrMint(transaction)
-            else -> Timber.e("Unsupported transaction: $transaction")
-        }
-    }
-
-    private fun parseSwap(transaction: HistoryTransaction.Swap) {
-        view?.apply {
-            showDate(transaction.date.toDateTimeString())
-            showStatus(transaction.status)
-
-            val usdTotal = transaction.getReceivedUsdAmount()
-            val total = transaction.getFormattedAmount()
-            showSignature(transaction.signature)
-            showAddresses(transaction.sourceAddress, transaction.destinationAddress)
-            showAmount(total, usdTotal)
-            showFee()
-            showBlockNumber(transaction.getBlockNumber())
-            showSwapView(transaction.sourceIconUrl, transaction.destinationIconUrl)
-        }
-    }
-
-    private fun parseTransfer(transaction: HistoryTransaction.Transfer) {
-        view?.apply {
-            showDate(transaction.date.toDateTimeString())
-            showStatus(transaction.status)
-
-            showSignature(transaction.signature)
-            showAddresses(transaction.senderAddress, transaction.destination)
-
-            val usdTotal = transaction.getFormattedAmount()
-            val total = transaction.getFormattedTotal()
-            showAmount(total, usdTotal)
-            showFee()
-            showBlockNumber(transaction.getBlockNumber())
-            showTransferView(transaction.getIcon())
-        }
-    }
-
-    private fun parseBurnOrMint(transaction: HistoryTransaction.BurnOrMint) {
-        view?.apply {
-            showDate(transaction.date.toDateTimeString())
-
-            showSignature(transaction.signature)
-            showAddresses(transaction.senderAddress, transaction.destination)
-
-            val usdTotal = transaction.getFormattedAmount()
-            val total = transaction.getFormattedTotal()
-            showAmount(total, usdTotal)
-            showFee()
-            showBlockNumber(transaction.getBlockNumber())
-            showTransferView(transaction.getIcon())
-        }
-    }
-}
```

Comment:
Did you just rename this class and the git history broke here?

Reply:
yep, just renaming
you can see it in the commit history